diff --git a/node_modules/@nativescript/canvas-polyfill/DOM/Element.d.ts b/node_modules/@nativescript/canvas-polyfill/DOM/Element.d.ts
index 01870b9..f5f2a65 100644
--- a/node_modules/@nativescript/canvas-polyfill/DOM/Element.d.ts
+++ b/node_modules/@nativescript/canvas-polyfill/DOM/Element.d.ts
@@ -6,6 +6,7 @@ export declare class Element extends Node {
     nativeElement: any;
     private _width;
     private _height;
+    lang: string;
     _instance: any;
     constructor(tagName: any);
     get classList(): any;
diff --git a/node_modules/@nativescript/canvas-polyfill/DOM/Element.js b/node_modules/@nativescript/canvas-polyfill/DOM/Element.js
index 7faf366..fd58146 100644
--- a/node_modules/@nativescript/canvas-polyfill/DOM/Element.js
+++ b/node_modules/@nativescript/canvas-polyfill/DOM/Element.js
@@ -21,6 +21,9 @@ export class Element extends Node {
     get tagName() {
         return this.nodeName;
     }
+    get lang() {
+        return 'EN';
+    }
     getAttribute(key) {
         return this._instance?.getAttribute?.(key) ?? null;
     }
